recipes/wxmplot/yum_requirements.txt

```diff
@@ -0,0 +1,2 @@
+gtk2-devel
+libXxf86vm
```

Comment:
This file may be needed in the `wxutils` recipe dir too, as eventually, they will appear in separate feedstock repositories.

Reply:
We have it in both feedstocks:
- https://github.com/nsls-ii-forge/wxmplot-feedstock/blob/master/recipe/yum_requirements.txt
- https://github.com/nsls-ii-forge/wxutils-feedstock/blob/master/recipe/yum_requirements.txt

Reply:
Alright I'll add it.